components/Misc/CreateContactModal.tsx

```
@@ -53,50 +53,55 @@ const CreateContactModal = ({
     if (isVisible !== undefined) {
       setShowContactModal(isVisible);
     }
-  }, [isVisible]);useEffect(() => {
+  }, [isVisible]);
+  useEffect(() => {
     const fetchCompleteJobData = async () => {
       if (contactToEdit && contactToEdit.jobApplications) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)